tests/utils/node_launcher.go

```diff
@@ -0,0 +1,68 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package utils
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/kurtosis-tech/kurtosis/api/golang/engine/lib/kurtosis_context"
+)
+
+const (
+	isPartitioningEnabled    = false
+	enclaveIdPrefix          = "test"
+	avalancheStarlarkPackage = "github.com/kurtosis-tech/avalanche-package"
```

Comment:
Is this running a static version of AvalancheGo/Subnet-EVM? For CI, we would want to build a Docker image of AvalancheGo that has the VM binary present in the plugin directory and launch a node running that otherwise it seems that this will not actually be testing the latest code.

Reply:
I've added image building as a step in the CI test and I pass the built image in the avalanche package now using the `:test` tag